common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -73,8 +76,21 @@
             // Failed to access the address field.
             addressField = null;
         }
+
         logger.debug("java.nio.Buffer.address: {}", addressField != null? "available" : "unavailable");
 
+        Constructor<?> directBufferConstructor;
+        try {
+            directBufferConstructor = direct.getClass().getDeclaredConstructor(long.class, int.class);
+            directBufferConstructor.setAccessible(true);
+        } catch (Throwable t) {
+            directBufferConstructor = null;
```

Comment:
we not do for others so I just wanted to keep it consistent. I mean even with the exception there is not much we can do.